.github/workflows/pull_request_release.yml

```diff
@@ -22,6 +24,9 @@ jobs:
           NODE_ENV=development npm ci
           git config --local user.email "one.amex@aexp.com"
           git config --local user.name "OneAmexBot"
+          npm run release:changelog
+          git add .
+          git commit -m "docs(changelog): add changelog"
```

Comment:
could this be dynamic and include the version ? ` git commit -m "docs(changelog): v5.0.1"` if not 
`git commit -m "docs(changelog): update"` as the changelog already exists

Reply:
That would so cool, i couldn't find a way to obtain. Maybe i could use this https://github.com/americanexpress/one-app/blob/master/scripts/generate-changelog.js